flaml/model.py

```diff
@@ -2270,6 +2270,7 @@ def fit(self, X_train, y_train, budget=None, **kwargs):
         from pytorch_forecasting.metrics import QuantileLoss
         import tensorboard as tb
 
+        self._kwargs = kwargs

```

Comment:
This works, but it'll keep the structure of the code more consistent if you just edit line 2291 from `gpus=self._kwargs.get("gpu_per_trial", [0])` to `gpus=kwargs.get("gpu_per_trial", [0])` since for this objects we used `kwargs.get()`. Thank you!

Reply:
Sure. I have removed the assignment and changed self.kwargs to just kwargs as you suggested.